test/attr/attr_objc_resilience.swift

```diff
@@ -4,6 +4,7 @@
 // RUN: %target-swift-frontend -typecheck -verify %s -I %t
 
 // REQUIRES: objc_interop
+// UNSUPPORTED: OS=iosmac
```

Comment:
Could you provide some explanation for this change? Is it unsupported (which means we need to write a different test that tests this feature), or is it like an XFAIL -- and then we need a tracking bug?

Reply:
This does not need to be in this update, going to remove it. 
```suggestion

```